### ISSUE TITLE
it doesn't work

### ISSUE THREAD

**Issue by milanjya** [lulop-k/kurento-rtsp2webrtc#12]
it doesn't work when rtsp like 'rtsp://ip:554/pag://ip:cameraid:0:MAIN:TCP'.
test in hiKvison iVMS-8600.
who can help me?

**Comment by Kaminight** [comment]
Yes it is works, but you should add in you url, the user and password. for example : rtsp://user:pass@ip:port...

I m not a expert, but I just working with hikvision.

**Comment by yangang1899** [comment]
> Yes it is works, but you should add in you url, the user and password. for example : rtsp://user:pass@ip:port...
> 
> I m not a expert, but I just working with hikvision.

Hi, ask a base question. Does this demo must be installed on ubuntu system?

**Comment by yangang1899** [comment]
> it doesn't work when rtsp like 'rtsp://ip:554/pag://ip:cameraid:0:MAIN:TCP'.
> test in hiKvison iVMS-8600.
> who can help me?
me too. the html reply me 'reconnect to server'.
did you fix your question? can you help me?

**Comment by SXH0212** [comment]
@Kaminight 
I test IP cam in VLC, it is connect  (rtsp://user:pass@ip:port)
but 
![image](https://user-images.githubusercontent.com/80878020/114402203-2de1a000-9bd6-11eb-943a-56099ef99e2a.png)

**Comment by sraza295** [comment]
@SXH0212 
Did you fixed this issue
